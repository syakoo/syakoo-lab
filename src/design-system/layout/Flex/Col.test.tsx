import { testStories } from "@/test/testStories";

import * as stories from "./Col.stories";

describe("Col", () => {
  testStories(stories);
});
