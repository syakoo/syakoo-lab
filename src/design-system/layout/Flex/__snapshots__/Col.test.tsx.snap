// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col renders Col_ 1`] = `
<body>
  <div>
    <div
      class="Flex_flexStyle__15ivi84i Flex_flexStyle_align_stretch__15ivi84u Flex_flexStyle_variants_align_stretch__15ivi849 Flex_flexStyle_direction_column__15ivi84k Flex_flexStyle_gap_100__15ivi84y Flex_flexStyle_variants_gap_100__15ivi84d Flex_flexStyle_justify_flexStart__15ivi84l Flex_flexStyle_variants_justify_flexStart__15ivi840"
    >
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: 260px; height: 260px; display: flex; align-items: center; justify-content: center;"
      >
        260px
         x 
        260px
      </div>
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: 140px; height: 140px; display: flex; align-items: center; justify-content: center;"
      >
        140px
         x 
        140px
      </div>
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: 200px; height: 200px; display: flex; align-items: center; justify-content: center;"
      >
        200px
         x 
        200px
      </div>
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: auto; height: 100px; display: flex; align-items: center; justify-content: center;"
      >
        auto
         x 
        100px
      </div>
    </div>
  </div>
</body>
`;

exports[`Col renders ColWithItem 1`] = `
<body>
  <div>
    <div
      class="Flex_flexStyle__15ivi84i Flex_flexStyle_align_stretch__15ivi84u Flex_flexStyle_variants_align_stretch__15ivi849 Flex_flexStyle_direction_column__15ivi84k Flex_flexStyle_gap_100__15ivi84y Flex_flexStyle_variants_gap_100__15ivi84d Flex_flexStyle_justify_flexStart__15ivi84l Flex_flexStyle_variants_justify_flexStart__15ivi840"
    >
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: 260px; height: 260px; display: flex; align-items: center; justify-content: center;"
      >
        260px
         x 
        260px
      </div>
      <div
        class="Flex_flexItemStyle__15ivi8419 Flex_flexItemStyle_align_center__15ivi841f Flex_flexItemStyle_variants_align_center__15ivi8415"
      >
        <div
          style="border-radius: 8px; background: rgb(64, 74, 84); width: 100%; height: 140px; display: flex; align-items: center; justify-content: center;"
        >
          100%
           x 
          140px
        </div>
      </div>
      <div
        class="Flex_flexItemStyle__15ivi8419 Flex_flexItemStyle_align_stretch__15ivi841h Flex_flexItemStyle_variants_align_stretch__15ivi8417"
      >
        <div
          style="border-radius: 8px; background: rgb(64, 74, 84); width: 100%; height: 200px; display: flex; align-items: center; justify-content: center;"
        >
          100%
           x 
          200px
        </div>
      </div>
      <div
        style="border-radius: 8px; background: rgb(64, 74, 84); width: auto; height: 100px; display: flex; align-items: center; justify-content: center;"
      >
        auto
         x 
        100px
      </div>
    </div>
  </div>
</body>
`;
